abci/multiplexer.go

```diff
@@ -273,6 +284,10 @@ func (m *Multiplexer) FinalizeBlock(_ context.Context, req *abci.RequestFinalize
 	// update the app version
 	m.lastAppVersion = resp.ConsensusParamUpdates.GetVersion().App
 
+	if m.lastAppVersion != m.activeVersion.AppVersion {
```

Comment:
need to think a bit more about this tomorrow, as m.lastAppVersion could be different than m.activeVersion.AppVersion, in the case of celestia, this is different from 1,2,3 as we use only one binary. i do not think it matters, but want to iron it out.

Reply:
using > then should do it actually.